lib/common/xml.c

```diff
@@ -1497,10 +1523,19 @@ sort_change_obj_by_position(gconstpointer a, gconstpointer b)
     return 0;
 }
 
+/*!
+ * \internal
+ * \brief Apply a version 2 patchset to an XML node
+ *
+ * \param[in,out] xml       XML to apply patchset to
+ * \param[in]     patchset  Patchset to apply
+ *
+ * \return Standard Pacemaker return code
+ */
 static int
 xml_apply_patchset_v2(xmlNode *xml, xmlNode *patchset)
```

Comment:
Likewise here, with `apply_v2_patchset`.

Reply:
updated

Reply:
whoops, jumped the gun on that one, there was already an apply_v2_patchset() :(

Reply:
never mind, i misread, the commits i just pushed are fine